### PR TITLE
for #94: Share button for Code Snippets game

### PR DESCRIPTION
**Pull request by chucker**
Minor note: there should be more spacing below the share button. I'm a bit confused how to do this in Blazorise (I _only_ want padding at the bottom).

### CODE REVIEW

**Comment by ricardoboss** [comment]
> I only want padding at the bottom

`<Div Padding="Padding.Is1.FromBottom">`

iirc

File: Prolangle/Components/ShareCodeSnippetButton.razor [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	[Parameter]
+	public required IReadOnlyList<ILanguage> RevealedLanguages { get; set; }
+
+	Snackbar _snackbar = null!;
+	string _snackbarMessage = "";
+
+	private async Task OnClicked()
+	{
+		var shareContent = $"Found today's #prolangle language in {"attempt".ToQuantity(RevealedLanguages.Count)}!\n\n" +
+		                   "Try it yourself at https://prolangle.ricardoboss.de";
```
Comment (ricardoboss) [inline Prolangle/Components/ShareCodeSnippetButton.razor:30]: Maybe directly link to the snippets game?

**Comment by chucker** [comment]
> > I only want padding at the bottom
> 
> `<Div Padding="Padding.Is1.FromBottom">`
> 
> iirc
> 
> Also note that `Div` is capitalized to use the Blazorise component instead of the default HTML element.

Ah, it comes _after_ the `Is1`. Got it.

File: Prolangle/Components/ShareCodeSnippetButton.razor (hunk 1 above)
Comment (chucker) [reply Prolangle/Components/ShareCodeSnippetButton.razor]: yep

**Comment by chucker** [comment]
See #110.

**Pull request merged**